package.json

```diff
@@ -8,6 +8,8 @@
   },
   "scripts": {
     "build": "ember build",
+    "docs": "yuidoc",
+    "docs-deploy": "yuidoc && ghp-import -r origin -p -b gh-pages docs/",
```

Comment:
Fun discovery: you can run `npm run <command_name> --arg1 value ...`, and actually pass arguments in after an npm command.

So there are ways to use a script, but make it configurable. :)